packages/frontend-core/src/components/grid/cells/RelationshipCell.svelte

```diff
@@ -259,14 +261,16 @@
       on:wheel={e => (focused ? e.stopPropagation() : null)}
     >
       {#each value || [] as relationship}
-        {#if relationship.primaryDisplay}
+        {#if relationship[primaryDisplay] || relationship.primaryDisplay}
```

Comment:
Is this needed because the value for user column types is a different shape than a normal relationship, so we don't have a `primaryDisplay` key? I assume so, but just double checking.

Reply:
Exactly. We were passing this primaryDisplay but it was never used, as the data was enriched at some point before. In our case we don't have it, so we need to make use of this field.
It could be safer inverting the operation, and always checking the `relationship.primaryDisplay` value first. But I think it's more intuitive to use the requested value in the component.